src/main/java/com/alex/eshop/service/UserService.java

```diff
@@ -37,4 +47,40 @@ public UserRepresentation getUserRepresentation(String userUuid) {
         return userRepresentation;
     }
 
+    public UserDTO getUser(String userUuid) {
+        UserRepresentation userRepresentation = keycloakClientFactory.getInstance()
+                .realm(applicationProperties.getKeycloak().getRealm())
+                .users().get(userUuid).toRepresentation();
+
+        UserDTO userDTO = new UserDTO();
+
+        userDTO.setUserUuid(userRepresentation.getId());
+        userDTO.setFirstName(userRepresentation.getFirstName());
+        userDTO.setLastName(userRepresentation.getLastName());
+        userDTO.setEmail(userRepresentation.getEmail());
+        userDTO.setRoles(userRepresentation.getRealmRoles());
+
+        return userDTO;
+    }
+
+    public Response createUser(UserRegisterDTO userRegisterDTO) {
+        UsersResource usersResource = keycloakClientFactory.getInstance().realm(applicationProperties.getKeycloak().getRealm()).users();
+
+        if (!userRegisterDTO.getPassword().equals(userRegisterDTO.getConfirmPassword())) {
+            throw new InvalidData("Invalid password");
+        }
+
+        CredentialRepresentation credentials = createPasswordCredentials(userRegisterDTO.getPassword());
+
+        UserRepresentation userRepresentation = new UserRepresentation();
+
+        userRepresentation.setUsername(userRegisterDTO.getUserName());
+        userRepresentation.setCredentials(Collections.singletonList(credentials));
+        userRepresentation.setFirstName(userRegisterDTO.getFirstName());
+        userRepresentation.setLastName(userRegisterDTO.getLastName());
+        userRepresentation.setEmail(userRegisterDTO.getEmail());
+        userRepresentation.setEnabled(true);
+
+        return usersResource.create(userRepresentation);
```

Comment:
createUser() is not finished.
`usersResource.create(userRepresentation)` will return `Response` type. It contains status code.
If status 201 - then everything is fine, user is created. If status 409 - then error.